manifest.json

```
@@ -1,33 +1,36 @@
 {
   "name": "Web Vitals",
   "version": "1.1.1",
-  "manifest_version": 2,
+  "manifest_version": 3,
   "description": "Measure metrics for a healthy site",
   "homepage_url": "https://web.dev/vitals/",
   "icons": {
     "128": "icons/vitals128w.png",
     "256": "icons/vitals256w.png",
     "512": "icons/vitals512w.png"
   },
-  "browser_action": {
+  "action": {
     "default_icon": "icons/default256w.png",
     "default_title": "Web Vitals",
     "default_popup": "src/browser_action/popup.html"
   },
   "options_page": "src/options/options.html",
-  "permissions": [
-    "tabs",
-    "storage",
-    "activeTab",
+  "host_permissions": [
     "http://localhost/",
     "http://*/*",
     "https://*/*",
     "https://www.googleapis.com/pagespeedonline/v5/runPagespeed?*"
```

Can we collapse everything into this single pattern?

https://developer.chrome.com/docs/extensions/mv3/match_patterns/

```suggestion
    "*://*/*"
```